### DISCUSSION TITLE
Is it supported combination? `strimzi/kafka:0.19.0-kafka-2.5.0` with `debezium-connector-postgres-1.4.1.Finalplugin` ?

### DISCUSSION THREAD

**Discussion by tomasz-zylka-fluke** [strimzi/strimzi-kafka-operator#4426] [Q&A]
Hi @scholzj 
we are using `strimzi/kafka:0.19.0-kafka-2.5.0` together with `debezium-connector-postgres-1.4.1.Finalplugin`.

Is this anyhow not recommended setup? By any chance are you aware of some major issues with it?

**Comment by scholzj** [comment]
Hi Tomasz ... I'm not aware of any issues. But to be honest I usually use Debezium only for demos etc. So I might not know about all problems. So it is more that I did not hear from anyone else about any issues.
